.circleci/config.yml

```diff
@@ -0,0 +1,23 @@
+jobs:
+  build:
+    docker: 
+      - image: circleci/node:10.17
+
+    branches:
+      only:
+        - master
+        - develop
+
+    steps:
+      - checkout
+      - add_ssh_keys:
+          fingerprints:
+            - "db:11:ad:b2:ad:8c:4a:e0:bd:32:f8:29:18:91:68:0a"
+      - run: |
+          if [ "$CIRCLE_BRANCH" = "master" ]; then
+            scp -oStrictHostKeyChecking=no -r $(pwd)/* $SERVER_USER@$SERVER_IP:~/cuhacking-api
```

Comment:
We should only be copying what we need to copy for the server to function. We specifically don't want to copy the node_modules folder.

That would be
- controllers
- model
- routes
- swagger.json
- server.js
- package.json

*(I believe that's everything)*

Then we also want to make sure we're old folders (node_modules, controllers, model, routes) so we don't have any lingering files when we copy them over.

To help with all these commands I suggest writing a bash script on the droplet.

Reply:
We're not copying a node_modules folder because I'm running the yarn command on the server, not on the build agent. Regardless, I've changed it to first remove, then selectively copy over the files we need.